html_sanitizer/sanitizer.py

```diff
@@ -25,7 +23,31 @@ def sanitize_href(href):
     return "#"
 
 
-def normalize_overall_whitespace(html):
+typographic_whitespace_names = [
+    "NO-BREAK SPACE",
+    "EN QUAD",
+    "EM QUAD",
+    "EN SPACE",
+    "EM SPACE",
+    "THREE-PER-EM SPACE",
+    "FOUR-PER-EM SPACE",
+    "SIX-PER-EM SPACE",
+    "FIGURE SPACE",
+    "PUNCTUATION SPACE",
+    "THIN SPACE",
+    "HAIR SPACE",
+    "NARROW NO-BREAK SPACE",
+    "MEDIUM MATHEMATICAL SPACE",
+    "IDEOGRAPHIC SPACE"
+]
+
+typographic_whitespace = "".join({unicodedata.lookup(n)
+    for n in typographic_whitespace_names})
```

Comment:
Minor nit: This does not look as if it was formatted by black. Run `tox` inside the repo to run `black` and `flake8` on the code.

Reply:
Ok, did it.